File: php/examples/02_app_management.php
<?php

require_once __DIR__ . '/../vendor/autoload.php';

use UIID\SDK\UIIDClient;

echo "=== 02. Application Management Workflow (PHP) ===\n\n";

echo "[App Mgmt] Available PHP SDK Methods:\n";
echo " - \$client->onboardApplication('My New App', 'https://myapp.com/callback')\n";
echo " - \$client->revokeApplication('app_123')\n\n";

File: python/examples/02_app_management.py
from uiid_client import UIIDClient

def main():
    print("=== 02. Application Management Workflow (Python) ===\n")

    client = UIIDClient()
    print("[App Mgmt] Methods available in Python SDK:")
    print(" - client.onboard_application(name='My App', redirect_uri='https://myapp.com/cb')")
    print(" - client.revoke_application(app_id='app_123')\n")

if __name__ == "__main__":
    main()
